resources/ui/adw/prefs.ui

```diff
@@ -214,39 +214,132 @@ SPDX-License-Identifier: GPL-3.0-or-later
     </child>
   </object>
 
+
   <object class="AdwPreferencesPage" id="general-prefs">
+    <property name="margin-top">20</property>
+    <property name="margin-bottom">20</property>
+    <property name="margin-start">20</property>
+    <property name="margin-end">20</property>
+
     <child>
-      <object class="AdwPreferencesGroup" id="effects-group">
-        <property name="margin-top">20</property>
-        <property name="margin-bottom">20</property>
-        <property name="margin-start">20</property>
-        <property name="margin-end">20</property>
-        <property name="title" translatable="yes">Effect Selection</property>
-        <property name="description" translatable="yes">If multiple are selected, one is chosen randomly.</property>
-
-        <child type="header-suffix">
-          <object class="GtkBox">
-            <property name="valign">center</property>
-            <style>
-              <class name="linked" />
-            </style>
+      <object class="AdwPreferencesGroup" id="">
+        
```

Comment:
Some things are kind of weird here. I think having a preferences group inside a preferences group is not desirable. I think you can simply move the title and description to the outer group and remove the inner group.

Add this:

```suggestion
      <object class="AdwPreferencesGroup">
         <property name="title" translatable="yes">Effect Selection</property>
         <property name="description" translatable="yes">If multiple are selected, one is chosen randomly.</property>
         
```

And completely remove the preferences group `header-text`.